src/pytypes/onefuzztypes/models.py

```diff
@@ -568,6 +568,7 @@ class NodeCommandEnvelope(BaseModel):
 
 class Node(BaseModel):
     timestamp: Optional[datetime] = Field(alias="Timestamp")
```

Comment:
```suggestion
    # Last modification time of node metadata.
    timestamp: Optional[datetime] = Field(alias="Timestamp")
```

Reply:
@nharper285, disregard this suggestion, since we're just inheriting it from Azure Table, and commenting further is actually more confusing than helpful (there's nothing special about it).